### ISSUE TITLE
support EC200N-CN-LA?

### ISSUE THREAD

**Issue by ChunxingJin** [waybyte/platform-logicrom#15]

**Comment by ajaybhargav** [comment]
yes, support can be added, do you have core firmware for EC200N-CN-LA?

**Comment by CharlesJin6** [comment]
> yes, support can be added, do you have core firmware for EC200N-CN-LA?

Unfortunately, I don't have an official firmware, but I do have an EC200N-CN-LA, which is the ASR1603 chip (based on Cortex-R5) in it. I don't know how to dump it.

**Comment by ajaybhargav** [comment]
here is the build for EC200N-CN-LA, Please test and let us know.

[Logicrom_ASR1603_EC200NCNLA_v0.2.0.zip](https://github.com/waybyte/platform-logicrom/files/12392719/Logicrom_ASR1603_EC200NCNLA_v0.2.0.zip)
